doc/rst/dfilemaker.1.rst

```diff
@@ -4,31 +4,20 @@ dfilemaker
 SYNOPSIS
 --------
 
-**dfilemaker <nitems> <nlevels> <maxflen>**
+**dfilemaker <destination_path>**
 
 DESCRIPTION
 -----------
 
-dfilemaker creates a random directory tree with files having random data
-that is useful for testing.
+dfilemaker creates a number of random directory tree with subdirectories and
+files of various sizes, that is useful for testing.
 
-Files and directories are created in the current working directory where the tool is executed.
-dfilemaker takes three positional parameters:
+Directory trees are created under destination_path, which must be a directory,
+where the tool is executed.
```

Comment:
Are these conflicting, or am I misunderstanding?
"under destination_path" or "where the tool is executed"?

Reply:
Oops, some of #614 leaked in accidentally.  I'll fix it.

Reply:
Actually, I left this for now so if you approve #614 and I merge it, I don't have to re-do what I removed.  I just won't merge this yet, in case you or Elsa have concerns with adding the parent_path argument.